roles/osg/templates/osg-deployer.j2

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+docker run --rm --net=host \
```

Comment:
Does this need "--net=host"? I think it's probably OK if it does, just wondering.